content/_data/events/scale15x.adoc

```diff
@@ -0,0 +1,6 @@
+---
+title: "Scale 15x"
+date: "2017-03-02 thru 03-05T09:00:00"
```

Comment:
alrighty, the date has been corrected. I also changed the file name since i'll need to create a file for each day, ergo need diff file names (I hope my understanding is correct :o))